src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingRequestToCsvMapper.java

```diff
@@ -20,6 +20,9 @@ public HearingRequestForCsv toHearingRequestForCsv(CaseHearingRequestEntity requ
                 requestEntity.getCaseUrlContextPath(),
                 requestEntity.getPublicCaseName()
             ));
+        if (isError == true) {
+            hearingRequestForCsv.setErrorDescription(requestEntity.getHearing().getErrorDescription().toString());
```

Comment:
boolean flag is not needed as AC01B in Jira says that  No Error Description values should be displayed as blank

Reply:
Boolean flags have been removed in latest commit and no error description is displayed as blank as needed for AC01B